pytorch_lightning/trainer/trainer.py

```diff
@@ -565,6 +565,7 @@ def __init__(
 
         # init logger flags
         self.logger: Optional[LightningLoggerBase]
+        self.loggers: Optional[List[LightningLoggerBase]]
```

Comment:
This doesn't need to be Optional. It can simply be an empty list in case no loggers are configured.